WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -5605,6 +5605,7 @@
 				B7A94351C1ADC31EA528B895 /* [CP] Embed Pods Frameworks */,
 				CE1445302188ED0300A991D8 /* Zendesk Strip Frameworks */,
 				59DF5B32D7C07692EC8B4C59 /* [CP] Copy Pods Resources */,
+				095040D72655531C001D08FA /* Check for nested frameworks */,
```

Comment:
Shameless plug for suggestion out of the scope of this PR. When having multiple custom run script build phases, I find it useful to track them in a dedicated ["main" script](https://mokacoding.com/blog/better-build-phase-scripts/#group-multiple-scripts-in-a-master-script) and only run that one.

Just leaving it out there...

Reply:
That's a nice article but I'm having trouble seeing how it would apply here, given that we have only that single script having to be run at the very very end of the list of Build phases and especially after the embed frameworks phase — so isolated from any other script we might otherwise have in the list of Build Phases?

Reply:
Oh I see. Fair enough. I saw the "Zendesk Strip Framework" in the diff and also noticed the call to Swiftlint in the project. But that's true: grouping all the scripts into a single one does force them to execute all at the same point of the build phase, which might not be desirable.

Nevermind 👍